modules/persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure_{context}"]
+= Dynamic provisioning across subscriptions for Azure File
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as an Azure identity in one subscription (call it Subscription A)
+
+* Access to another subscription (call it Subscription B) in the same tenant with the cluster
+
+* Logged in to the Azure CLI
+
+.Procedure
+To use Azure File dynamic provisioning across subscriptions:
+
+. Record the Azure identity (service principal or managed identity) by running the following applicable commands. The Azure identity is needed in a later step:
++
+* If using the _service principal_ as the Azure identity when installing the cluster:
++
+[source,terminal]
+----
+$ sp_id=$(oc -n openshift-cluster-csi-drivers get secret azure-file-credentials -o jsonpath='{.data.azure_client_id}' | base64 --decode)
+
+$ az ad sp show --id ${sp_id} --query displayName --output tsv
+----
++
+* If using the _managed identity_ as the Azure identity when installing the cluster:
++
+[source,terminal]
+----
+$ mi_id=$(oc -n openshift-cluster-csi-drivers get secret azure-file-credentials -o jsonpath='{.data.azure_client_id}' | base64 --decode)
+
+$ az identity list --query "[?clientId=='${mi_id}'].{Name:name}" --output tsv
+----
+
+. Grant the Azure identity (service principal or managed identity) permission to access the resource group in another Subscription B where you want to provision the Azure File share by doing one of the following:
+
+* Run the following Azure CLI command:
++
+[source,terminal]
+----
+az role assignment create \
+  --assignee <object-id-or-app-id> \
+  --role <role-name> \
+  --scope /subscriptions/<subscription-id>/resourceGroups/<resource-group>/providers/Microsoft.Storage/storageAccounts/<storage-account-name>
+----
++
+Where:
++
+`<object-id-or-app-id>`: The service principal or managed identity that you obtained from the previous step, such as `sp_id` or `mi_id`.
++
+`<role-name>`: Role name. Contributor or your own role with required permissions.
++
+`<subscription-id>`: Cross subscription B ID.
```

Comment:
do we need to call it "cross", it's just subscription B ID isn't it?

Reply:
Yes. I've changed it.